treeherder/perf/models.py

```diff
@@ -244,14 +247,18 @@ class PerformanceAlertSummary(models.Model):
                                       default=1)  # Bugzilla
 
     def update_status(self):
-        autodetermined_status = self.autodetermine_status()
+        own_alerts = PerformanceAlert.objects.filter(summary=self)
+        alerts = (own_alerts | PerformanceAlert.objects.filter(related_summary=self))
```

Comment:
I'm curious why you're using the bitwise 'or' here.

Reply:
I think this could be written like so:
```
queryset = PerformanceAlert.objects
alerts = (queryset.filter(summary=self) | queryset.filter(related_summary=self))

Reply:
> I'm curious why you're using the bitwise 'or' here.

At some point I considered I needed to compute parent's `first_triaged` timestamp based on its direct children. Turned out this wasn't a correct assumption, but I didn't revert to the old code back. Thanks for spotting this!